### PR TITLE
Update to fix Rundeck 2.7.x installation

### PR DESCRIPTION
**Pull request by isuftin**
https://github.com/criteo-cookbooks/rundeck-server/issues/43

- Added Test Kitchen suite for Rundeck 2.7.x
- Updated Java installation to default to version 8
- Added Rundeck CLI installation recipe
- Added attribute to fix H2 installation/execution
- Added attributes to properly configure Rundeck 2.7.x CLI
- Added documentation 
- Fixes some Rubocop warnings
- Version bump
- Update changelog
- Some foodcritic fixes
- Updated default installation version in attributes to 2.7.x

### CODE REVIEW

File: recipes/install_cli.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Cookbook: rundeck-server
+# Recipe:   install_cli
+# Description: This should be ran if using Rundeck 7.x+
+# See: http://rundeck.org/docs/upgrading/index.html#cli-tools-are-gone
+# Note: This must be Java 8 or greater
+include_recipe 'java' if node['rundeck_server']['install_java']
+
+package 'rundeck-cli'
```
Comment (kamaradclimber) [inline recipes/install_cli.rb]: shouldn't we allow version to be configured

**Comment by kamaradclimber** [comment]
also, please rebase to get test passing (I've just update the .travis.yml to make test from forks passing correctly)

**Comment by isuftin** [comment]
@kamaradclimber Updated as per request, please let me know if you need more on this

**Comment by kamaradclimber** [comment]
thanks a lot!

**Pull request merged**